src/components/pagination/pagination.jsx

```diff
@@ -0,0 +1,172 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import injectSheet from 'react-jss';
+import { Icon } from '../../';
+import Page from './page';
+import Stepper from './stepper';
+import Ellipsis from './ellipsis';
+import styles from './pagination-styles';
+
+const getRandomString = () =>
```

Comment:
This will break on SSR (Math.random() will evaluate differently on server and client). Can we make this deterministic somehow?